tests/e2e/e2e_test.go

```diff
@@ -49,6 +50,25 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 	s.Require().Equal(concentratedPool.GetTickSpacing(), tickSpacing)
 	s.Require().Equal(concentratedPool.GetPrecisionFactorAtPriceOne(), sdk.NewInt(precisionFactorAtPriceOne))
 	s.Require().Equal(concentratedPool.GetSwapFee(sdk.Context{}), sdk.MustNewDecFromStr(swapFee))
+
+	// Initialize 2 more nodes
+	node2, err := chainA.GetNodeAtIndex(1)
+	s.Require().NoError(err)
+	node3, err := chainA.GetNodeAtIndex(2)
+	s.Require().NoError(err)
+
+	minTick, maxTick := cl.GetMinAndMaxTicksFromExponentAtPriceOne(sdk.NewInt(precisionFactorAtPriceOne))
+
+	// Create 2 positions for node1: overlap together, overlap with 2 node3 positions)
+	node1.CreateConcentratedPosition(initialization.ValidatorWalletName, "[-1200]", "400", fmt.Sprintf("1000%s", denom0), fmt.Sprintf("1000%s", denom1), 0, 0, poolID)
```

Comment:
What matters is using different addressed to create positions from.

We would like to have positions created from the same addresses as well as some different addresses. Overlapping and non-overlapping